project-base/web/assets/styleguide/images/icon_full.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Generator: Adobe Illustrator 16.0.4, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
Is this necessary here? And in other svgs?

Reply:
It is not necessary. Deleted.